<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1706</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Imagen</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Imagen original</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="85,15">
              <item>
               <widget class="MplWidget" name="w_entrada" native="true"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>100</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QListWidget" name="lw_entrada"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_agregar_entrada">
                  <property name="text">
                   <string>Agregar...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_eliminar_entrada">
                  <property name="text">
                   <string>Eliminar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>#P</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="sp_proy1">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>999</number>
                    </property>
                    <property name="singleStep">
                     <number>1</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_radon">
                  <property name="text">
                   <string>Radón</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Reconstrucción</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="85,15">
              <item>
               <widget class="MplWidget" name="w_salida" native="true"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>100</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QListWidget" name="lw_salida"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_eliminar_salida">
                  <property name="text">
                   <string>Eliminar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QComboBox" name="cb_metodo">
                  <item>
                   <property name="text">
                    <string>Python</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ART</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>FBP</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Fourier</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>GC</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cb_filtro">
                  <item>
                   <property name="text">
                    <string>Sin filtro</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ram-Lak</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Shepp-Logan</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Coseno</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Hamming</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Hann</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_radon_inv">
                  <property name="text">
                   <string>Inversa Radón </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Sinograma</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Sinograma</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="85,15">
              <item>
               <widget class="MplWidget" name="w_sinograma" native="true"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>100</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QListWidget" name="lw_sinogramas"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_agregar_sinograma">
                  <property name="text">
                   <string>Agregar...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_eliminar_sinograma">
                  <property name="text">
                   <string>Eliminar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_radon_inv_2">
                  <property name="text">
                   <string>Inversa Radón</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8" stretch="5,95">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>#P</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sp_proy2">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0">
              <item>
               <widget class="MplWidget" name="w_proy" native="true"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>FFT</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>FFT Original</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MplWidget" name="w_fft" native="true"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_15" stretch="0,4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>FFT de la Reconstrucción</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MplWidget" name="w_fft_2" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Filtros</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0">
            <item>
             <widget class="QLabel" name="label_filtro">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Filtro</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="85,0">
              <item>
               <widget class="MplWidget" name="w_filtro" native="true"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <spacer name="verticalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>100</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QComboBox" name="cb_filtro_2">
                  <item>
                   <property name="text">
                    <string>Sin filtro</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ram-Lak</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Shepp-Logan</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Coseno</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Hamming</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Hann</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12" stretch="5,95">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>#P</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sp_proy_filt">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="0">
              <item>
               <widget class="MplWidget" name="w_proy_filt" native="true"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>widgets.mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
